src/fake_useragent/fake.py

```diff
@@ -10,27 +10,22 @@
 class FakeUserAgent:
     def __init__(
         self,
-        cache=True,
-        use_cache_server=True,
-        path=settings.DB,
+        use_external_data=False,
+        tmp_path=settings.DB,
```

Comment:
`tmp_path` doesnt feel like a good name for this.  The file provided in this arg isnt a temporary file.  It is more like a cache file...?

Reply:
True you are right